README.md

```diff
@@ -26,29 +25,34 @@ You can get the watchtower-operator cli prebuilt, or build from source
     ./build
     ```
 
+
 ## How to use
-Once you have the watchtower-operator installed, you can directly use the exectable -
+To build the CLI, use the following command - 
```

Comment:
This would be redundant instruction, build was already mentioned in the `Installation` section

Reply:
Okay I will keep the old line as it is